rlog_test.go

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package rlog
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+)
+
+var logfile string
+var removeLogfile bool = false
+var fixedLogfileName bool = true
+
+// setup is called at the start of each test and prepares a new log file. It
+// also returns a new configuration, which can be used by this test.
+func setup() rlogConfig {
+	if fixedLogfileName {
+		logfile = "/tmp/rlog-test.log"
+	} else {
+		logfile = fmt.Sprintf("/tmp/rlog-test-%d.log", time.Now().UnixNano())
```

Comment:
I'd added the 'fixedLogfileName' flag as a means to test my own tests during development. In fact, I forgot to set this back to 'false'. I'll also add a comment for those flags, explaining that this is for internal testing. Otherwise, I'd have to edit lines of code in my tests while developing.